.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,102 @@
+name: End-to-end test
+
+on:
+  push:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment to deploy to"
+        type: environment
+        required: true
+  workflow_run:
+    workflows: [Deployment]
```

Comment:
I don't think we want this to trigger on every single successful Deployment run. We really only want this to run when deployment is successful to environments containing dummy data. As things stand currently I believe we would introduce dummy data into a prod environment upon a successful deploy to prod.